urdfenvs/albert_reacher/resources/albert_robot.py

```diff
@@ -13,9 +13,18 @@ def __init__(self):
         self._wheel_distance = 0.494
 
     def set_joint_indices(self):
-        self._urdf_joints = [10, 11, 14, 15, 16, 17, 18, 19, 20]
-        self._robot_joints = [24, 25, 8, 9, 10, 11, 12, 13, 14]
-        self._castor_joints = [22, 23]
+        wheel_joint_names = ["wheel_right_joint", "wheel_left_joint"] 
+        mmrobot_joint_name = ["mmrobot_joint" + str(i) for i in range(1,8)] 
+        self._joint_names = ( 
+            wheel_joint_names 
+            + mmrobot_joint_name 
+        ) 
+        robot = URDF.load(self._urdf_file) 
+        self._urdf_joints = [] 
+        for i, joint in enumerate(robot.joints): 
+            if joint.name in self._joint_names: 
+                self._urdf_joints.append(i) 
+        self.get_indexed_joint_info()
```

Comment:
Can't we move that to a central function? Currently, these lines are in all the robot files.